frontends/concrete-python/examples/sha1/sha1.md

```diff
@@ -0,0 +1,307 @@
+# SHA1 Computation with Modules
+
+We take the example of a SHA1 computation to show how
+[Modules](https://docs.zama.ai/concrete/compilation/modules) can be useful in Concrete. SHA1 is a
+deprecated broken hash function: we didn't take this example for its security, but rather as it's a
+pedagogical example of how Modules can be useful.
+
+The SHA1 code is given [here](sha1.py). Execution times of the different functions are given in the
+final section.
+
+We made our example by forking
+[https://github.com/ajalt/python-sha1](https://github.com/ajalt/python-sha1) by
+[AJ Alt](https://github.com/ajalt), and then made lot of modifications to have an FHE implementation
+and corresponding tests.
+
```

Comment:
> We take the example of a SHA1 computation to show how
> [Modules](https://docs.zama.ai/concrete/compilation/modules) can be useful in Concrete. SHA1 is a
> deprecated broken hash function: we didn't take this example for its security, but rather as it's a
> pedagogical example of how Modules can be useful.
> 
> The SHA1 code is given [here](sha1.py). Execution times of the different functions are given in the
> final section.
> 
> We made our example by forking
> [https://github.com/ajalt/python-sha1](https://github.com/ajalt/python-sha1) by
> [AJ Alt](https://github.com/ajalt), and then made lot of modifications to have an FHE implementation
> and corresponding tests

**I propose to change to :**
This document demonstrates the use of modules in Concrete through a SHA1 computation example. SHA1 is a deprecated and broken hash function; we use it here for pedagogical purposes, not for its security.

The SHA1 code is available [here](sha1.py). Execution times for the different functions are provided in the final section. We created our example by forking [python-sha1](https://github.com/ajalt/python-sha1) by [AJ Alt](https://github.com/ajalt) and made extensive modifications to implement FHE and corresponding tests.

Reply:
ok, took it almost as you said, just changed "made extensive modifications to implement SHA1 in FHE"